src/Events/Integrations/Provider.php

```diff
@@ -33,6 +33,12 @@ public function register() {
 		$this->container->register( Plugins\Colbri_Page_Builder\Provider::class );
 		$this->container->register( Plugins\Event_Tickets\Provider::class );
 		$this->container->register( Plugins\Elementor\Provider::class );
+		add_action(
+			'tribe_plugins_loaded',
+			function () {
+				$this->container->register( Plugins\TEC_Tweaks_Extension\Provider::class );
+			}
+		);
```

Comment:
Cant we use `$this->container->register_on_action()`

Reply:
Oh, good call. I am blind.